precice-config.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0"?>
+
+<precice-configuration>
+
+  <log enabled="1">
+    <sink filter="%Severity% > debug" />
+  </log>
+  
+  <solver-interface dimensions="2">
+    
+    <!-- Data fields that are exchanged between the solvers -->
+    <data:scalar name="Pressure"/>
+    <data:scalar name="CrossSectionLength"/>
+
+    <!-- A common mesh that uses these data fields -->
+    <mesh name="Fluid-Nodes">
```

Comment:
The meshes should now be names `Fluid-Mesh-Nodes` etc (we want the `-Mesh` part there, for consistency).